src/components/DND/DragDropContext.tsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import { Draggable } from "components/DND/Draggable";
+import Droppable from "components/DND/Droppable";
+
+const findScrollParents = (
+  element: HTMLElement | null
```

Comment:
Can you update the column dragging so that when you start dragging the column, it centers in the middle of the header rather than the center of the column?